lib/rules/inline-link-to.js

```diff
@@ -1,20 +1,42 @@
 'use strict';
 
+const { builders: b } = require('ember-template-recast');
+
 const Rule = require('./base');
 
 const message = 'The inline form of link-to is not allowed. Use the block form instead.';
 
 module.exports = class InlineLinkTo extends Rule {
   visitor() {
     return {
-      MustacheStatement(node) {
+      MustacheStatement(node, { parentNode, parentKey }) {
         if (node.path.original === 'link-to') {
-          this.log({
-            message,
-            line: node.loc && node.loc.start.line,
-            column: node.loc && node.loc.start.column,
-            source: this.sourceForNode(node),
-          });
+          if (this.mode === 'fix') {
+            let titleNode = node.params[0];
+
+            let newBody;
+            if (titleNode.type === 'SubExpression') {
+              newBody = b.mustache(titleNode.path, titleNode.params, titleNode.hash);
+            } else if (titleNode.type === 'StringLiteral') {
+              newBody = b.text(titleNode.value);
+            }
+
+            let index = parentNode[parentKey].indexOf(node);
+            parentNode[parentKey][index] = b.block(
+              node.path,
+              node.params.slice(1),
+              node.hash,
+              b.blockItself([newBody])
+            );
+          } else {
+            this.log({
+              message,
+              line: node.loc && node.loc.start.line,
+              column: node.loc && node.loc.start.column,
+              source: this.sourceForNode(node),
+              isFixable: true,
```

Comment:
At the moment it is only fixable if `node.params[0]` is either a `SubExpression` or `StringLiteral`, can you update this to only be true when we can _for sure_ fix it, or add support for the other types of literal?

Reply:
done